scipy/interpolate/fitpack2.py

```diff
@@ -1644,8 +1738,23 @@ class RectSphereBivariateSpline(SphereBivariateSpline):
 
     See Also
     --------
-    RectBivariateSpline : bivariate spline approximation over a rectangular
-        mesh
+    BivariateSpline :
+        to create a bivariate spline on a plane grid
```

Comment:
Looking up the docs, the docstring of BivariateSpline, https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.BivariateSpline.html, says explicitly 

> Base class for bivariate splines. ... This class is meant to be subclassed, not instantiated directly.

IMO, it'd be better to have this in the SeeAlso sections instead, here and elsewehere.

Reply:
I use "a base class for bivariate splines" for one line comment in SeeAlso. PTAL.